extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -133,6 +133,11 @@ export const selectPublishOption = localize('selectPublishOption', "Select where
 export const publishToExistingServer = localize('publishToExistingServer', "Publish to existing server");
 export const publishToDockerContainer = localize('publishToDockerContainer', "Publish to new server in a container");
 export const enterPortNumber = localize('enterPortNumber', "Enter SQL server port number or press enter to use the default value");
+export const serverPortNumber = localize('serverPortNumber', "SQL server port number");
+export const serverPassword = localize('serverPassword', "SQL Server admin password");
+export const confirmServerPassword = localize('confirmServerPassword', "Confirm SQL Server admin password");
+export const baseDockerImage = localize('baseDockerImage', "Base SQL Server Docker image");
+export const publishTo = localize('publishTo', "Publish To");
```

Comment:
Based on the screenshot, I think this might make more sense as "Publish Destination" or "Publish Target" since both options start with "Publish to..."